tests/Oryx.BuildImage.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -252,6 +252,46 @@ public void BuildsApplication_ByDynamicallyInstalling_IntoCustomDynamicInstallat
                 result.GetDebugInfo());
         }
 
+        [Fact]
+        public void GeneratesScript_AndBuilds_WithPackageDir()
+        {
+            // Arrange
+            var version = "3.6.9";
+            var appName = "flask-app";
+            var installationDir = $"{BuildScriptGenerator.Constants.TemporaryInstallationDirectoryRoot}/python/{version}";
+            var volume = CreateSampleAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var appOutputDir = "/tmp/app-output";
+            var packagesDir = ".python_packages/lib/python3.7/site-packages";
```

Comment:
the package dir is `.python_packages/lib/python3.6.9/site-packages` for this test?

Reply:
Yes, it is the same directory structure that the az functions team tried to use. I just wanted to keep it the same for this test.